lib/components/search/helper.ts

```diff
@@ -21,6 +25,19 @@ export const search = (keyword: string, local: string): SearchResults => {
   return data
 }
 
+export const groupResults = (data: SearchResults) => {
+  return data.reduce<SearchResultGroup[]>((acc, item) => {
+    const title = item.group || 'General'
+    const group = acc.find(group => group.title === title)
+    if (!group) {
+      acc.push({ title, items: [item] })
+    } else {
+      group.items.push(item)
```

Comment:
There is actually a side effect here, the `group` is updated by reference, a better way would be:
```
return source.map(item => ({ ...item, title }))
```